recipe/meta.yaml

```
@@ -1,19 +1,21 @@
 {% set name = "pytorch-lightning" %}
-{% set version = "1.9.5" %}
+{% set version = "2.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 925fe7b80ddf04859fa385aa493b260be4000b11a2f22447afb4a932d1f07d26
+  sha256: b37a3677a65cda920260de3899ea9dfdd221b0a2d838a0439e31bd927efa61f4
 
 build:
   number: 0
-  # TODO: the upper bound is because of PyTorch issue with dataclass, see https://github.com/Lightning-AI/lightning/issues/15614
-  skip: True  # [py<37 or py>310]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # the upper bound is because of PyTorch issue with dataclass, see https://github.com/Lightning-AI/lightning/issues/15614
+  skip: True  # [py<37]
+  # skipping py311 for win and linux-ppc64le because for now dep pytorch is not supported on such platformsyou 
+  skip: True  # [py>310 and (win or (linux and ppc64le))]
```

```suggestion
  skip: True  # [py>310 and (linux and ppc64le)]
```
We do not have python 3.11 support only on ppc64le because of pytorch

---

ok